File: commands/utility/avatar.go
package utility

import (
  "github.com/sapphire-cord/sapphire"
  "fmt"
)

// View someone's avatar.
// Usage: [@user]
// Aliases: av, pfp
func Avatar(ctx *sapphire.CommandContext) {
  user := ctx.Author

  if ctx.HasArgs() {
    user = ctx.Arg(0).AsUser()
  }

  ctx.BuildEmbed(sapphire.NewEmbed().
    SetTitle(fmt.Sprintf("%s's Avatar", user.Username)).
    SetColor(0xDFAC7C).
    SetImage(user.AvatarURL("2048")).
    SetAuthor(ctx.Author.Username, ctx.Author.AvatarURL("256")))
}

File: commands/utility/avatar.js
const Command = require("../../structures/Command.js");
const { MessageEmbed } = require("discord.js");

class Avatar extends Command {
  constructor(...args) {
    super(...args, {
      description: "Grab someone's avatar.",
      aliases: ["av", "pfp"],
      usage: "avatar [@user]"
    });
  }

  async run(ctx, [userArg]) {
    const user = await this.verifyUser(ctx, userArg, true);
    
    return ctx.reply(new MessageEmbed()
      .setTitle(`${user.username}'s avatar`)
      .setAuthor(user.tag, user.displayAvatarURL({ size: 64 }))
      .setColor(0x9590EE)
      .setImage(user.displayAvatarURL({ size: 2048, format: "png", dynamic: true })));
  }
}

module.exports = Avatar;
